entries/s/sdfcu.org.json

```diff
@@ -0,0 +1,20 @@
+{
+  "State Department Federal Credit Union": {
+    "domain": "sdfcu.org",
+    "additional-domains": [
+       "sdfcuib.org"
```

Comment:
Are you sure [sdfcuib.org](https://sdfcuib.org/) is being used?

Reply:
I checked this out earlier. There is no page on the root path, but they have some other paths set up, like [this one](https://www.sdfcuib.org/dbank/live/).

Reply:
Yeah, that domain is used on some pages, like that.

I can take it out if it doesn't make sense to include it.

Reply:
The additional domains are particularly useful for password managers and browser extensions to be able to associate a site to its login/password. Since the `sdfcuib.org` is used to offer login forms, it should definitely be included here.